src/backend/users/views.py

```diff
@@ -55,6 +55,16 @@ def get(self, request, **kwargs):
         return response
 
 
+class DeleteToken(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, **kwargs):
```

Comment:
I think this would be more natural as a `POST` -- `GET`s are usually more about retrieving information, whereas a `POST` is more of a "do-something" verb.

Reply:
Got it, will change!